akka-http-core/src/test/scala/akka/http/impl/engine/client/RedirectTestSpec.scala

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.http.impl.engine.client
+
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.model.headers.{ Location, RawHeader }
+import akka.http.scaladsl.settings.{ ClientAutoRedirectSettings, ClientConnectionSettings }
+import akka.http.scaladsl.{ Http, TestUtils }
+import akka.stream.scaladsl._
+import akka.stream.testkit.Utils
+import akka.stream.{ ActorMaterializer, ActorMaterializerSettings }
+import akka.testkit.AkkaSpec
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class RedirectTestSpec extends AkkaSpec {
+  implicit val materializer = ActorMaterializer(ActorMaterializerSettings(system).withFuzzing(true))
```

Comment:
fuzzing is enabled by default (see reference.conf in test scope), you can simply `ActorMaterializer()`